lib/ansible/modules/cloud/amazon/emr_cluster_facts.py

```diff
@@ -0,0 +1,317 @@
+#!/usr/bin/python
+# Copyright: Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'certified'}
+
+
+DOCUMENTATION = '''
+---
+module: emr_cluster_facts
+short_description: Get information about an Amazon EMR cluster
+description:
+    - Module gets information about an Amazon EMR cluster
+version_added: "2.6"
+requirements: [ boto3 ]
+author:
+    - "Aaron Smith (@slapula)"
+options:
+    id:
+        description:
+            - ID of the EMR cluster.
+        required: false
```

Comment:
Looking at the boto3 docs, it looks like cluster ID is required for the describe_cluster call you are making. list_clusters would be a call you could use if no cluster_id is provided.

required: false is also the default, so if this remains to be true you can remove this line.